v3/src/components/case-card/card-view.scss

```diff
@@ -30,7 +30,7 @@
     color: #0592af;
     cursor: pointer;
     display: block;
-    font-size: 10px;
+    font-size: 12px;
```

Comment:
Is this the right size? The spec says 10px but then Dan said it was too small and that it "should match the font size of the rest of the text on the data card view" which is 11px. Did you and/or Dan decide it should be slightly larger? To me it looks a little too big now which is why I double checked, but maybe that's just me.

Reply:
Good catch - will change to 11px.